ehcache-impl/src/main/java/org/ehcache/impl/internal/store/heap/SimpleBackend.java

```diff
@@ -28,43 +28,39 @@
 import java.util.Random;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.BiFunction;
-import java.util.function.Supplier;
 
 /**
  * Simple passthrough backend, no key translation
  */
 class SimpleBackend<K, V> implements Backend<K, V> {
 
-  private volatile EvictingConcurrentMap<K, OnHeapValueHolder<V>> realMap;
-  private final Supplier<EvictingConcurrentMap<K, OnHeapValueHolder<V>>> realMapSupplier;
```

Comment:
This field will move up to the OnHeapStore and become a Supplier of Backend types.

Reply:
I have move the map Supplier to level up at - `OnHeapStore.java`